<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroductionPage</class>
 <widget class="QWidget" name="IntroductionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="prevButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>560</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>   The topic of our application is Machine Learning.  With AI models like ChatGPT becoming popular in the past year, there's also been a lot of discussion on the abilities of the technology.  Will AI become sentient?  Will ChatGPT take over the world?
  While there is some truth to the common saying that these models are &quot;black boxes&quot;, it can help to understand what they are really doing behind the web-based interface we see.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QFrame" name="introBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="introLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>37</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Introduction</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>21</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>560</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scatter-Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>   Our application presents an intuitive idea for how Machine Learning models work, and what they are really doing behind the scenes.  To show this, we will focus on one specific model: Linear Regression.

   The next page will demonstrate what Linear Regression is used for, and you will get to see it doing it's job in real-time.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
